src/components/project/handlers/set-department-id.handler.ts

```diff
@@ -48,9 +48,9 @@ export class SetDepartmentId implements IEventHandler<SubscribedEvent> {
         MATCH ()-[:departmentId]-(departmentIdPropertyNode:Property)
         WHERE departmentIdPropertyNode.value STARTS WITH $departmentIdPrefix
```

Comment:
Line 49 is where the problem actually was I think. Since `$departmentIdPrefix` was a number and not a string I think it wasn't matching the existing dept ids. However, since the same value was being inserted as a JS template literal var on 53 and surrounded by quotes, that bit was working.